bit/find_missing_number.py

```diff
@@ -0,0 +1,19 @@
+def find_missing_number(nums):
+    """Returns the missing number from a sorted list of integers
+    in O(n) time and space.
+
```

Comment:
I guess, we should mention this in the comments. 'Sorted list of integers' does not explain it.